.changeset/weak-coats-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-linguist': patch
+---
+
+Updated `README.md` to removed images with refresh button as this feature was not implemented in the current version
```

Comment:
If it's a READMe only update, you likely don't need a changeset. That triggers a release, which isn't necessary in this case?

Reply:
My understanding to update this on NPM is was needed as well

Reply:
Yep! A changeset is needed for README updates to be reflected on NPM